src/transcript-plugin.tsx

```diff
@@ -267,8 +268,14 @@ export class TranscriptPlugin extends KalturaPlayer.core.BasePlugin {
       showTranscript: <Text id="transcript.show_plugin">Show Transcript</Text>,
       hideTranscript: <Text id="transcript.hide_plugin">Hide Transcript</Text>
     };
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore
     this._transcriptIcon = this.upperBarManager!.add({
-      label: 'Transcript',
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-ignore
+      displayName: 'Transcript',
+      ariaLabel: 'Transcript',
+      order: 30,
```

Comment:
why set order value if the order already defined in https://github.com/kaltura/playkit-js-ui-managers/blob/271049a2c5fd6f95d2642aa9b46693e1716cfd90/src/ui-managers.ts#L22

Reply:
Central config in UI-managers removed and decentralised, now ui-managers is decoupled from any plugin and should not know them 
https://github.com/kaltura/playkit-js-ui-managers/pull/51/files#diff-7e81688e2aaa4578e6b335ce46c47dcd7e5bbe4ced6b38368436761faf5cd1e8L16